bundle/manifests/cert-manager-operator.clusterserviceversion.yaml

```diff
@@ -250,7 +250,7 @@ metadata:
     features.operators.openshift.io/token-auth-aws: "true"
     features.operators.openshift.io/token-auth-azure: "true"
     features.operators.openshift.io/token-auth-gcp: "true"
-    olm.skipRange: '>=1.15.1 <1.16.0'
+    olm.skipRange: '>=1.15.1 <1.16.1'
```

Comment:
This OLM upgrade path is unlike other z-stream
releases to help users in <=1.15.1 to entirely avoid OCPBUGS-56758.
See [this](https://issues.redhat.com/browse/CM-604?focusedId=27441517&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-27441517) for the upgrade path strategy.